packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -65,7 +65,8 @@ class Scrollable extends StatefulWidget {
   /// ScrollableState scrollable = Scrollable.of(context);
   /// ```
   static ScrollableState of(BuildContext context) {
-    return context.ancestorStateOfType(const TypeMatcher<ScrollableState>());
+    final _ScrollableScope widget = context.inheritFromWidgetOfExactType(_ScrollableScope);
```

Comment:
At some point (not this patch) we might want to provide a flag here to say if you want to subscribe or not, since otherwise just trying to call Scrollable.of() to scroll will hook you up.
(I think some of our .of()s do this already?)